libs/portals/admin/signature-collection/src/screens-parliamentary/index.tsx

```diff
@@ -30,6 +35,18 @@ const ParliamentaryRoot = ({
   const navigate = useNavigate()
   const { collection, allLists } = useLoaderData() as ListsLoaderReturn
 
+  const [searchTerm, setSearchTerm] = useState('')
+
+  const { data, loading } = useSignatureCollectionSignatureLookupQuery({
+    variables: {
+      input: {
+        collectionId: collection?.id,
+        nationalId: searchTerm,
+      },
+    },
+    skip: searchTerm.replace(/[^0-9]/g, '').length !== 10,
```

Comment:
This will strip a string of everything that's not a number. This correct?
I assume it is for the skip but just checking to make sure :)